src/app/components/filebrowseruss/filebrowseruss.component.ts

```diff
@@ -257,6 +257,7 @@ export class FileBrowserUSSComponent implements OnInit, OnDestroy {//IFileBrowse
           res => {
             this.snackBar.open(`Renamed: ${oldName} to ${nameFromNode}`,
               'Dismiss', { duration: 5000,   panelClass: 'center' });
+            this.USSRenameEvent.emit(this.lastRightClickEvent);
```

Comment:
IMO: Should be `this.lastRightClickEvent.node` like the others.
Typically, we emit the node object instead of the whole event which is very hefty, because we already know what the event was and now we only need the data that the FE provides

There could be a usecase where the Editor or some app could try to manipulate a node HTML element directly, but I don't think it would be good to slow things down for this small edgecase. In the future, we could add optional args

Reply:
If you see why this instead could be bad, lmk